src/core_modules/capture-core/metaDataMemoryStoreBuilders/common/helpers/dataElement/unsupportedMultiText/handleMultiTextPreV41.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import log from 'loglevel';
+import { errorCreator, hasAPISupportForFeature, FEATURES } from 'capture-core-utils';
+import { dataElementTypes, DataElement } from '../../../../../metaData';
+
+const isMultiTextPreV41 = (type, minorServerVersion: number) =>
+    type === dataElementTypes.MULTI_TEXT && !hasAPISupportForFeature(minorServerVersion, FEATURES.multiText);
+
+export const handleMultiTextPreV41 = (dataElement: DataElement, minorServerVersion: number) => {
```

Comment:
NIT: remove `PreV41` from the functions name and use a non-specific spelling instead. Perhaps something like `handleMultiTextFallback` and `isMultiTextSupported` can work.

Reply:
I agree the suffix (PreV41) wasn't optimal. Really hard to find a descriptive name though. Going for `handleMultiTextInUnsupportedBackendVersions` and `isMultiTextInUnsupportedBackendVersion`. Long, but at least you get a sense of what it is about.